auth-api/src/auth_api/exceptions/errors.py

```diff
@@ -140,6 +140,7 @@ class Error(Enum):
     PATCH_INVALID_ACTION = "PATCH_INVALID_ACTION", HTTPStatus.BAD_REQUEST
     SERVICE_UNAVAILABLE = "3rd party application unavailable", HTTPStatus.SERVICE_UNAVAILABLE
     NOT_AUTHORIZED_TO_PERFORM_THIS_ACTION = "Not authorized to perform this action", HTTPStatus.FORBIDDEN
+    NSF_CLIENT_CANNOT_CREATE_ACCOUNT = "Clients with NSF status cannot create new accounts.", HTTPStatus.FORBIDDEN
```

Comment:
NSF or Suspended, I think they are two different cases - might be a good idea to make this more generic

Clients that are NSF or SUSPENDED cannot create new accounts

Let me show you a screenshot 

![image](https://github.com/user-attachments/assets/2d696f32-26d4-4f9f-8df6-f56fd3228fac)